api/parts/data/fetch.js

```diff
@@ -1089,6 +1089,35 @@ fetch.fetchDataEventsOverview = function(params) {
     }
 };
 
+/**
+* Get top events data
+* @param {params} params - params object
+**/
+
+fetch.fetchDataTopEvents = function(params) {
+    const {
+        qstring: { app_id, period, limit }
+    } = params;
+    const collectionName = "top_events" + crypto.createHash("sha1").update(app_id).digest("hex");
+    common.db.collection(collectionName).findOne({period}, function(error, result) {
```

Comment:
We might need index here on app_id + period, if this will be used single collection for all apps